python/ray/dashboard/subprocesses/module.py

```diff
@@ -197,8 +200,14 @@ def run_module(
         backup_count=config.logging_rotate_backup_count,
     )
 
+    stderr_filename = module_logging_filename(
+        module_name, config.logging_filename, is_stderr=True
+    )
+    err_file = open_log(os.path.join(config.log_dir, stderr_filename), unbuffered=True)
+    configure_log_file(err_file, err_file)
```

Comment:
Ideally stdout should go to dashboard_xxx.out file but let's merge this as it is to match the existing behavior.